Project.toml

```diff
@@ -15,7 +15,7 @@ MLJModelInterface = "1.3"
 ScientificTypesBase = "2.2.0"
 ScientificTypes = "2.2.2"
 TextAnalysis = "0.7.3"
-julia = "1.3"
+julia = "1.5"
```

Comment:
Are you still getting a fail? Can you please re-instate 1.3 so I can investigate.  This sounds fishy. If 1.3 is failing it should only be because we are using a newer julia feature.

Reply:
it's still failing - CI is still running 1.3 because I see now you have to update the proper file in `.github/workflows`.  I can confirm that the code I posted above (`test_dict2 = Dict{NGram, Int}(("a", "b") => 1, ("c", "d", "e") => 2)`) fails in Julia 1.3.  Whatever issue is causing Julia to crash got fixed/changed sometime between Julia 1.4 and Julia 1.5 (or perhaps was fixed in Julia 1.5 which is the first version I tried where it did not fail).

Reply:
Thanks! I'll look into the this today.